src/datanode/src/instance.rs

```diff
@@ -315,11 +315,21 @@ pub(crate) async fn new_object_store(store_config: &ObjectStoreConfig) -> Result
         ObjectStoreConfig::Oss { .. } => new_oss_object_store(store_config).await,
     };
 
+    // Don't enable retry layer when using local file backend.
+    let object_store = if !matches!(store_config, ObjectStoreConfig::File(..)) {
+        object_store.map(|object_store| object_store.layer(RetryLayer::new().with_jitter()))
+    } else {
+        object_store
+    };
+
     object_store.map(|object_store| {
         object_store
-            .layer(RetryLayer::new().with_jitter())
             .layer(MetricsLayer)
-            .layer(LoggingLayer::default())
+            .layer(
+                LoggingLayer::default()
+                    // Print expected error only in DEBUG level.
+                    .with_error_level(Some(log::Level::Debug)),
```

Comment:
The default level is warning, changing it debug will generate more logs. To cut redundant logs I think you can only change it to error level

Reply:
Hi, the documentation for `LoggingLayer` may be misleading. The log level refers to the level used by `LoggingLayer` to emit logs, which is done through `log!(level, "xxxx")`. I will improve this part of docs later.